src/libs/getKeyboardType/index.ios.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
This file is not needed. If we've added `index.android.js`, then for other platforms it'll fallback to `index.js`